### PR TITLE
Replace DeviceDefault themes with material 3 themes

### PR DESCRIPTION
**Pull request by thagikura**
Because the behaviour of using DeviceDefault theme is not guaranteed
and now the material components has the themes for widgets.

### CODE REVIEW

File: AppWidget/build.gradle [hunk 1]
```diff
@@ -17,15 +17,15 @@
...
     }
     repositories {
         google()
         mavenCentral()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:7.0.0'
-        classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:1.5.10"
+        classpath 'com.android.tools.build:gradle:7.0.4'
+        classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:1.5.30"
```
Comment (yaraki) [inline AppWidget/build.gradle:28]: Why not the same version as above?

Comment (yaraki) [reply AppWidget/build.gradle:28]: * about kotlin plugin being '1.5.30' not '1.6.10'

Comment (thagikura) [reply AppWidget/build.gradle]: Done.

**Review by yaraki — APPROVED** [review]

**Comment by thagikura** [comment]
Thanks Yuichi. Is it possible to have a look, too @marcelpinto ?

**Comment by marcelpinto** [comment]
> Thanks Yuichi. Is it possible to have a look, too @marcelpinto ?

LGTM. Just wondering if we still need the night folders if we are using the DayNight theme.

**Review by marcelpinto — APPROVED** [review]

**Comment by thagikura** [comment]
> LGTM. Just wondering if we still need the night folders if we are using the DayNight theme.

Oh good point. Checked with light and dark themes, and it looks like night themes for night variants can be removed, done.

**Review by marcelpinto — APPROVED** [review]

**Comment by thagikura** [comment]
Sorry, forgot to merge this PR.
Merging.

**Pull request merged**